### DISCUSSION TITLE
Proxmox Installations Problems

### DISCUSSION THREAD

**Discussion by 4358haltern** [aaronwmorris/indi-allsky#1615] [Q&A] [answered]
hey
After I installed it in a Proxmox container, I always get the following message after the installation (see screenshot). Restarting the system and restarting the script didn't help. can you help me❓

![Bildschirmfoto 2024-11-07 um 17 01 09](https://github.com/user-attachments/assets/2fba7071-873b-4d56-ab9e-8594da96793c)

**Comment by aaronwmorris** [comment]
The normal method of installing and managing indi-allsky relies heavily on systemd and DBUS.  With container environments, it is not unusual for this to not function correctly.

You may have to setup a VM in proxmox and then use docker within the VM.  There is a docker compose script to install and manage indi-allsky through docker, but I do not believe LXC containers have a composer platform to automate the install of software.

Or maybe you could run nested docker in the container.

**Reply by 4358haltern** [reply]
ok❗️ so moving to a vm❗️ no problem. can you recommend a guide

**Reply by aaronwmorris** [reply, marked as answer]
I just have the Docket page in my wiki.  What kind of camera are you using?

https://github.com/aaronwmorris/indi-allsky/wiki/Docker
